tests/attention/test_fmha_v2_prefill.py

```
@@ -844,6 +851,118 @@ def test_trtllm_fmha_v2_prefill(
+    device = torch.device("cuda")
+    dtype = torch.bfloat16
+    num_qo_heads, num_kv_heads, head_dim = 8, 2, 256
+    seq_len = 64
+
+    query = torch.randn(seq_len, num_qo_heads, head_dim, dtype=dtype, device=device)
+    kv = torch.randn(seq_len, 2, num_kv_heads, head_dim, dtype=dtype, device=device)
+    seq_lens = torch.tensor([seq_len], dtype=torch.int32, device=device)
+    cu_seqlens = torch.tensor([0, seq_len], dtype=torch.int32, device=device)
+    workspace = torch.zeros(8 * 1024 * 1024, dtype=torch.uint8, device=device)
+
+    with pytest.raises(ValueError, match="[Cc]hunked"):
```

Ruff (0.15.15)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)